index.html

```diff
@@ -171,6 +171,16 @@ <h4>2015</h4>
             </ul>
     </section>
 </div>
+<section class="contact-me">
+    <p class="contact-intro1">Contact me</p>
+    <p class="contact-intro2">If you have an application you <br>are interested in developing, a<br> feature that you need built or a <br>project that needs coding. I’d <br>love to help with it </p>
+    <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Kindly create a [Formspree](https://formspree.io/html/) account and add your id to this form action so you can receive user messages through this form. Your form is very similar to the template, but without your id, it is not useful.

Reply:
@leonardodiasb I have created the account but I don't see my id.

Reply:
@leonardodiasb I have created an account on Formspree and added my id to this form as requested